core/src/main/java/org/apache/calcite/sql/type/OperandTypes.java

```diff
@@ -1266,7 +1266,7 @@ public static SqlSingleOperandTypeChecker same(int operandCount,
   public static final SqlSingleOperandTypeChecker DIVISION_OPERATOR =
       NUMERIC_NUMERIC.or(INTERVAL_NUMERIC);
 
-  public static final SqlSingleOperandTypeChecker MINUS_OPERATOR =
+  public static final SqlSingleOperandTypeChecker  MINUS_OPERATOR =
```

Comment:
please remove the extra space, it shouldn't be there and it adds useless work for the reviewer.